kahuna/public/stylesheets/main.css

```diff
@@ -518,6 +518,15 @@ input.search-query__input {
     display: none;
 }
 
+.file-uploader__select-files {
+    padding: 5px 10px;
+}
+
+@media screen and (max-width: 1200px) {
```

Comment:
I wouldn't say so - it goes against our component based organisation.
If I were working on the upload button I want to open upload.css and edit everything in there.